calico_kubernetes/calico_kubernetes.py

```diff
@@ -14,9 +14,11 @@
 from logutils import configure_logger
 from pycalico import netns
 from pycalico.datastore import IF_PREFIX, DatastoreClient
+from pycalico.datastore_errors import DataStoreError
 from pycalico.util import generate_cali_interface_name, get_host_ips
 from pycalico.ipam import IPAMClient
-from pycalico.datastore_errors import PoolNotFound
+
+logger = logging.getLogger(__name__)
```

Comment:
Could you change "logger" to "_log" ?

It will standardize the naming with our other repos.